meson.build

```diff
@@ -40,6 +40,10 @@ endforeach
 # The debugedit program is a hard dependency
 debugedit = find_program('debugedit', version: '>= 5.0')
 
+# Require appstream with compose plugin installed
+appstreamcli = find_program('appstreamcli', version: '>= 0.15.0')
+appstreamcli_compose = run_command(appstreamcli, ['--help'], check: true)
```

Comment:
This doesn't check whether the compose plugin is installed. I think this should be changed to match the `configure.ac` check thusly:

```suggestion
appstreamcli_compose = run_command(appstreamcli, ['compose', '--help'], check: true)
```

Reply:
No this should be the other way around we specifically need compose

Reply:
Good catch, thanks

Reply:
Yes, we specifically need compose, and the Meson check currently doesn't check for it.

Reply:
Hmm, maybe the CLI tools should also ship pkg-config files, to make version and presence checking easier...

Reply:
it does ship pkg-config files

Reply:
![image](https://github.com/flatpak/flatpak-builder/assets/30698906/96fb051f-6938-4483-b159-10b2c6891c12)
on opensuse it ships under `AppStream-compose-devel`

Reply:
> both appstream and appstream compose ship pkg-config files

Yes, but for the library development files, not for the command-line utility. You can have the libraries without CLI...

Reply:
> > both appstream and appstream compose ship pkg-config files
> 
> Yes, but for the library development files, not for the command-line utility. You can have the libraries without CLI...

given we actually call [`appstreamcli compose`](https://github.com/flatpak/flatpak-builder/blob/8c98ddb8f5b2ee3a63265b984f4c46e434aab444/meson.build#L45) I think @barthalion is right we do need to check for the `appstreamcli compose` command vs the libs